settings_downloader_function/requirements.txt

```
@@ -2,7 +2,7 @@ google-api-python-client
 oauth2client
 google-cloud-bigquery
 google-auth
-google-analytics-admin
+git+https://github.com/googleapis/python-analytics-admin@main
```

@annecremin it looks like this repo has been archived and its functionality moved elsewhere. I think there are two options.

- Migrate to the google-cloud-python package which will require some code changes.
- Attempt to use the last-known stable version of python-analytics-admin (v0.22.0).

I would suggest we try the second option first. 
```suggestion
git+https://github.com/googleapis/python-analytics-admin@v0.22.0
```